### DISCUSSION TITLE
[4.x] Menu Seperator

### DISCUSSION THREAD

**Discussion by brianteeman** [joomla/joomla-cms#46998] [2. Cold Features]
### Steps to reproduce the issue
Create a menu module in the "menu" position
Create several menu items including one that is of the type "system/seperator"

![image](https://github.com/joomla/joomla-cms/assets/1296369/1f6bff24-39fa-41c3-b0ac-e3c0b38b9506)

### Expected result
the seperator is displayed as 
| menu title

### Actual result
If using either of the layouts from cassiopeia
|

If using wither of the  core layouts
menu title

**Comment by chmst** [comment]
I cannot imagine what you are expecting, could you share some design or screen? 
 
The intention was a vertical line in horizontal menus and a text (due to b/c) in vertical menus. 
You also can use an image or icon as separator.

**Comment by brianteeman** [comment]
The descriptions of the menu type says that it can display a seperator and a text label. That is what it did in j3. In this example the telephone number is the menu title

![image](https://github.com/joomla/joomla-cms/assets/1296369/2a49d26d-fd54-43db-ba98-ceb160d6b43c)

With the Cassiopeia overrides it will ignore the title
Without the overrides it will ignore the seperator |

**Comment by chmst** [comment]
Thank you. now I understand.

**Comment by drmenzelit** [comment]
The problem is that the standard menu doesn't have CSS of its own. Only a Javascript that does wrong things ... but that is another story ...

**Comment by drmenzelit** [comment]
Or maybe was the idea, that one can use icons or images as separator (if you don't display the text). As @chmst wrote on my (now closed) PR #45489 we should rewrite the code of the menu types
